docs/list-a-project.md

```diff
@@ -48,7 +47,7 @@ tags:
   - the
   - project
 upforgrabs:
-  name: up for grabs
+  name: Up-For-Grabs
```

Comment:
This change is not correct, as it's part of the project configuration and the casing here is important

Reply:
Good call!